Diagnostics/HealthChecker/DataCollection/ExchangeInformation/IISInformation/Get-IISWebSite.ps1

```diff
@@ -2,9 +2,22 @@
 # Licensed under the MIT License.
 
 function Get-IISWebSite {
-    $webSites = Get-WebSite
-    $bindings = Get-WebBinding
+    param(
+        [array]$WebSitesToProcess
+    )
+
     $returnList = New-Object 'System.Collections.Generic.List[object]'
+    $webSites = New-Object 'System.Collections.Generic.List[object]'
+
+    if ($null -eq $WebSitesToProcess) {
+        $webSites.Add((Get-WebSite))
```

Comment:
This can't be done this way. 

![image](https://user-images.githubusercontent.com/22776718/199996825-89416126-32fd-4d9f-a5f0-4e1916ada3c1.png)

`AddRange` must be used. 

![image](https://user-images.githubusercontent.com/22776718/199997066-b42cb5eb-8041-41f0-97f3-57b5f118d614.png)

Reply:
Copy/Past error. Good catch! Fixed.